Zend/zend_extensions.c

```diff
@@ -280,6 +280,40 @@ ZEND_API int zend_get_op_array_extension_handles(const char *module_name, int ha
 	return handle;
 }
 
+ZEND_API size_t zend_internal_run_time_cache_reserved_size() {
+	return zend_op_array_extension_handles * sizeof(void *);
+}
+
+ZEND_API void zend_init_internal_run_time_cache() {
+	size_t functions = zend_hash_num_elements(CG(function_table));
+	zend_class_entry *ce;
+	ZEND_HASH_MAP_FOREACH_PTR(CG(class_table), ce) {
+		functions += zend_hash_num_elements(&ce->function_table);
+	} ZEND_HASH_FOREACH_END();
+
+	size_t rt_size = zend_internal_run_time_cache_reserved_size();
+	if (rt_size) {
+		void *ptr = zend_arena_calloc(&CG(arena), functions, rt_size);
+		zend_internal_function *zif;
+		ZEND_HASH_MAP_FOREACH_PTR(CG(function_table), zif) {
+			if (!ZEND_USER_CODE(zif->type) && ZEND_MAP_PTR_GET(zif->run_time_cache) == NULL)
+			{
+				ZEND_MAP_PTR_SET(zif->run_time_cache, ptr);
+				ptr += rt_size;
```

Comment:
At least MSVC doesn't like void* arithmetic, i.e. it errors with [C2036](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2036?view=msvc-170). Same issue 8 lines below.

Reply:
thanks, changed to char * :-)